src/components/InvoiceTable/index.jsx

```diff
@@ -0,0 +1,52 @@
+import {useEffect, useState} from "react";
+
+function InvoiceTable({invoiceDetails}) {
+
+    const details = invoiceDetails.details;
+    const displayDetails = (item) => {
+        return (
+            <tr>
+                <td>{item.description}</td>
+                <td>{item.quantity}</td>
+                <td>{item.rate}</td>
+                <td>£{item.total}</td>
+            </tr>
+        )
+    }
+
+    const calculatedTotal = details.reduce((total, current) => total + Number(current.total), 0)
+    return (
+        <>
+            <div className="App">
+                <table className="table w-100">
+                    <thead>
+                    <tr>
+                        <th>Description</th>
+                        <th>Quantity</th>
+                        <th>Rate</th>
+                        <th>Total</th>
+                    </tr>
+                    </thead>
+                    <tbody className="table-group-divider border-secondary-subtle">
+                    {details.map(displayDetails)}
```

Comment:
I'm not 100% sure, but React might give you a warning here about needing to use a `key` prop for each table row. Check to make sure!